tee_supplicant/src/tee_supplicant.c

```diff
@@ -52,6 +68,87 @@ static int receive_request(const struct device *dev, struct tee_supp_msg *ts_req
 	return rc;
 }
 
+#define MEM_ID(id) ((id) ? (uint64_t)((struct tee_shm *)(id))->addr : 0)
```

Comment:
macro definition in the middle of file looks weird

Reply:
Moved to bottom of file